tests/fixtures/classesImport/expected.php

```diff
@@ -3,14 +3,25 @@
 declare(strict_types=1);
 
 use Blumilk\Codestyle\Configuration\Defaults\CommonRules;
-use Blumilk\Codestyle\Configuration\Defaults\LaravelPaths;
 
 class TestException extends Exception
 {
-    protected string $rules = LaravelPaths::class;
+    protected string $var = "Error";
+    protected string $class = \Error::class;
+    protected string $laravel = "LaravelPaths";
+    protected string $bar = \PhpCsFixer\Tokenizer\Tokens::class;
```

Comment:
These should be imported. Maybe you could manipulate with priorities of fixers?

Reply:
fixed